yangcatalog-ui/src/app/features/yang-module-details/yang-module-details.component.html

```diff
@@ -43,8 +43,15 @@ <h2>YANG Module Details</h2>
         <!--templates for several types of response content-->
         <ng-template #revisionTemplate let-data="data">
           <select class="form-select" (change)="onRevisionSelectChange($event)" formControlName="moduleRevision">
-            <option value="{{rev}}" *ngFor="let rev of infoData.revisions"
-              [selected]="rev === form.get('moduleRevision').value">{{rev}}</option>
+            <option value="{{rev.revision}}" *ngFor="let rev of infoData.revisions"
+              [selected]="rev.revision === form.get('moduleRevision').value">
+              <ng-container *ngIf="checkIfRatified(rev.revision)">
+                &#10003; {{rev.revision}}
+              </ng-container>
+              <ng-container *ngIf="checkIfRatified(rev.revision) == false">
+                &#10007; {{rev.revision}}
```

Comment:
Do not add &#10007; mark before revision if `is_rfc` == false.
In other words only add &#10003; icon if module is RFC.
Maybe you will need to add some empty character or space before so the text is aligned in dropdown